lib/to-schema.js

```diff
@@ -35,21 +35,26 @@ const transformPrimitive = (key, value) => {
   return "String";
 };
 
```

Comment:
Saving a Google, which I find myself doing for Regex :)`\W` - Matches any character that is not a word character from the basic Latin alphabet